src/operations/blas3/gemm_partial_local.hpp

```diff
@@ -177,8 +223,8 @@ class GemmPartial<input_t, output_t, DoubleBuffer, NbcA, NbcB, ClSize,
   SYCL_BLAS_INLINE void eval(local_memory_t scratch,
                              cl::sycl::nd_item<1> id) noexcept {
     /* Pointers to the scratch memory (lhs and rhs) */
-    auto scratch_ptr = scratch.localAcc.get_pointer().get();
-    auto rhs_scratch_ptr = scratch_ptr + rhs_scratch_offset;
+    value_t* scratch_ptr = scratch.localAcc.get_pointer().get();
```

Comment:
you don't need .get() at the end anymore. ComputeCpp implicitly convert multi_pointer to the pointer * since you remove the auto